### ISSUE TITLE
About the discriminator does not work

### ISSUE THREAD

**Issue by 2805413893** [taozh2017/HiNet#13]
Previously modified line 151 in the Hinet_SynthModel.py file loss_fake = criterion_GAN(self.discrimator(x_fake.detach()), fake)，But now the discriminator loss is very large and the generator loss is very small, so the network cannot be trained. I would like to ask you if you encounter the same problem, please guide me.
